.github/workflows/rspec.yml

```diff
@@ -73,5 +73,5 @@ jobs:
           curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
           chmod +x ./cc-test-reporter
           ./cc-test-reporter before-build
-          RUBYOPT='-W:no-deprecated -W:no-experimental' bundle exec rspec
+          RUBYOPT='-W:no-deprecated -W:no-experimental' bundle exec rspec spec/models/judge.rb:7
```

Comment:
do not merge this line :)

Reply:
Yes definitely not! Just for testing/debugging purposes :)